Improve German translations

ReneSchimmelpfennig opened the pull request:
Fix some spelling issues and non-idiomatic expressions

IWasForcedToGetThis commented:
The new, fixed translations are fine for the most part. The translation for "Check Update" should be reverted though since it's not very good and the new German translation for "Always Run" would translate to "Run By Default" which is not the point of the original English name. Everything else works for this commit.

ReneSchimmelpfennig commented:
> The new, fixed translations are fine for the most part. The translation for "Check Update" should be reverted though since it's not very good and the new German translation for "Always Run" would translate to "Run By Default" which is not the point of the original English name. Everything else works for this commit.

Thanks for your feedback, I implemented your suggestions in my pull request :)
